ckcp/openshift/base/deployment.yaml

```diff
@@ -38,10 +38,11 @@ spec:
           value: ckcp-ckcp.apps.domain.org
         resources:
           limits:
+            cpu: 1000m
```

Comment:
What is the purpose of the cpu limit?
Limit does not play any role in scheduling or resource reservation.
It could protect against a bad behavior of kcp but if kcp is not responding there is nothing left to protect.

Reply:
I'll remove this. Since CPU is compressible, limits only throttle the process. This doesn't really matter for development.

Reply:
Thanks!